README.md

```diff
@@ -85,4 +85,8 @@ docker-compose up -d
 ### Generate data
 
 After the local cluster is up and running, you can generate some data to work with, 
-using`go run -tags prd ./cmd/generate`.
\ No newline at end of file
+using`go run -tags prd ./cmd/generate`.
+
+### Run `ktea`
+
+Use `go run -tags dev cmd/ktea/main.go` to run `ktea` from the root of the repository.
```

Comment:
Maybe add that the dev tag  will simulate an artificial slow network by sleeping for 2 seconds when doing network IO. This way the loaders and spinners can be visually asserted.